packages/store/addon/-private/system/core-store.ts

```diff
@@ -1580,7 +1580,7 @@ abstract class CoreStore extends Service {
       typeof modelName === 'string'
     );
 
-    const resource = constructResource(modelName, ensureStringId(id));
+    const resource: ResourceIdentifierObject = constructResource(modelName, ensureStringId(id));
```

Comment:
Do we like to type the return value?  If no value, I can remove

Reply:
If it's not super clear, it's :+1: